Even Pair Sum Problem

mansi-9 opened the pull request:
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

Two positive integers A and B are given. Find the number of pairs of positive integers (X,Y) such that 1≤X≤A, 1≤Y≤B and X+Y is even.

Input
The first line of the input contains a single integer T denoting the number of test cases. The description of T test cases follows.
The first and only line of each test case contains two space-separated integers A and B.
Output
For each test case, print a single line containing one integer ― the number of valid pairs.
Constraints
1≤T≤1,000
1≤A,B≤109

Example Input
4
1 1
2 3
4 6
8 9
Example Output
1
3
12
36

### Checklist

- [yes ] I've read the contribution guidelines.
- [ yes] I've checked the issue list before deciding what to submit.
- [ yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#2752 issue i.e. GSSOC 2021 Even Pair Sum Problem. Added in cp folder of C-Plus-Plus folder.

mansi-9 committed:
resolved issue #2752 i.e.GSSOC 2021 Even Pair Sum Problem #2752.Added file in cp folder of C-Plus-Plus folder

bhav09 reviewed (commented):
Great work! 
However, would request a few amendments. Can you please add comments above the code segment and not alongside. And, please put I/O at the end (as you have done for complexity, already). Thanks

mansi-9 commented:
@bhav09 sure thing!!!

bhav09 commented:
@mansi-9, yes, please do the changes and commit. I will review the files and then merge them if it is absolutely fine.

mansi-9 committed:
#2752 did the reguired changes

Added comments above the code segment and has put I/O at the end .
#2752 EvenPairSum

bhav09 commented:
Great work!

mansi-9 commented:
thanks!!
